client/src/App.js

```diff
@@ -4,6 +4,10 @@ import { ListItem } from "./ListItem/ListItem";
 import logo from "./logo.svg";
 import "./App.css";
 
+require('dotenv').config();
```

Comment:
I dont think you need this line, you can automatically access the env variables anywhere in the app if they are formatted as REACT_APP_API_KEY :)